hazelcast/src/test/java/com/hazelcast/client/impl/spi/impl/ClientClusterServiceImplTest.java

```diff
@@ -59,7 +60,7 @@ public class ClientClusterServiceImplTest extends HazelcastTestSupport {
     @Test
     public void testMemberAdded() {
         LinkedList<Member> members = new LinkedList<>();
-        ClientClusterServiceImpl clusterService = new ClientClusterServiceImpl(mock(ILogger.class));
+        ClientClusterServiceImpl clusterService = new ClientClusterServiceImpl(mock(HazelcastClientInstanceImpl.class), mock(ILogger.class));
```

Comment:
I do not see any new test case in this class, why did you convert to mock classes?